bin/auto_hck

```diff
@@ -9,6 +9,13 @@ module AutoHCK
   run do
     cli = CLI.new
     cli.parse(ARGV)
+    session_path = if cli.test.latest_session
+                     "#{Config.read['workspace_path']}/latest"
+                   elsif cli.test.session
+                     cli.test.session
+                   end
+
+    cli = Session.load(session_path, cli) if session_path
```

Comment:
Pass `cli.common.workspace_path` instead of `cli` to contain all code dealing with the original `cli` here (including the code deriving `session_path`).